frontend/src/assets/css/breakpoint-variables.css

```diff
@@ -0,0 +1,7 @@
+:root {
+    --breakpoint-sm: 576px;
+    --breakpoint-md: 768px;
+    --breakpoint-lg: 992px;
+    --breakpoint-xl: 1200px;
+    --breakpoint-xxl: 1400px;
```

Comment:
```suggestion
    --sm: 576px;
    --md: 768px;
    --lg: 992px;
    --xl: 1200px;
```

4 breakpoint enough

Reply:
Done in commit [79c8d92](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/79c8d924c82ac336a7e505cad7d9b7de4a8f2db8)